pkg/skaffold/schema/latest/config.go

```diff
@@ -589,6 +589,9 @@ type Generate struct {
 	// RawK8s defines the raw kubernetes resources.
 	RawK8s []string `yaml:"rawYaml,omitempty" skaffold:"filepath"`
 
+	// RawK8sLifeCycleHooks describes a set of lifecycle hooks that are executed before and after every render.
+	RawK8sLifeCycleHooks RenderHooks `yaml:"rawYamlHooks,omitempty"`
```

Comment:
maybe we can simplify this by only having a single `RenderHooks` inside `Generate` that gets handled in the `render_mux.go` file:
```diff
- type GroupRenderer []Renderer
+ type GroupRenderer struct {
+  renderers []Renderer
+ hooks RenderHooks
+}
``` 

WDYT?

Reply:
I can make these changes but initially the idea was the keep the hooks per renderer philosophy similar to the hooks per deployer design vs hooks per render phase.  I will make this change and push them but it was unclear if this was acceptable or why similarly deploy doesn't have a top level hooks vs deployer level hooks

Reply:
I've updated the code here to have a render phase set of hooks at `manifests.hooks` vs per renderer hooks

Reply:
Have made the suggested change, updated the PR info to state:

Adds support for `render` phase Lifecycle hooks at 
`manifests.hooks`

NOTE: these hooks are pre/post the render phase and not per renderer